README.md

```diff
@@ -68,27 +68,61 @@ indicate if it is on sale or not.
 
 The following state attributes are available for this sensor:
 
-| attribute       | description                                      |
-| --------------- | ------------------------------------------------ |
-| box_art_url     | The URL for the box art of the game.             |
-| normal_price    | The normal price of the game.                    |
-| sale_price      | The sale price of the game.                      |
-| percent_off     | The percentage off of the normal price.          |
-| steam_id        | The Steam ID of the game.                        |
-| title           | The title of the game.                           |
-| review_desc     | The review description, e.g. `Mostly positive`   |
-| reviews_percent | The percentage of positive reviews for the game. |
-| reviews_total   | The total number of reviews.                     |
+| attribute       | description                                                    |
+| --------------- | -------------------------------------------------------------- |
+| title           | Title of the game                                              |
+| rating          | Reviews e.g. `Reviews: 92% (Very Positive)` &nbsp;**\***       |
+| price           | Price description of game &nbsp;**\***                         |
+| genres          | Genres of game e.g. `FPS, Action, First-Person` &nbsp;**\***   |
+| release         | Release date of game &nbsp;**\***                              |
+| airdate         | Date game was released (Unix timestamp format)                 |
+| normal_price    | Price                                                          |
+| percent_off     | Percentage off of the normal price                             |
+| review_desc     | Review description                                             |
+| reviews_percent | Percentage of positive reviews                                 |
+| reviews_total   | Total number of reviews                                        |
+| sale_price      | Sale price of the game                                         |
+| steam_id        | Steam ID of the game                                           |
+| box_art_url     | URL for the background 16:9 aspect ratio image                 |
+| fanart          | URL for the background 16:9 aspect ratio image &nbsp;**\***    |
+| poster          | URL for the background 3:4 image &nbsp;**\***                  |
+| deep_link       | Clickable hyperlink to game on Steam website &nbsp;**\***      |
+
+
+• _Attributes marked with **\*** are for **upcoming_media_card** only._
```

Comment:
This note probably isn't necessary as anyone could use any of the values for any automation.

Reply:
Agreed